catroid/src/androidTest/java/org/catrobat/catroid/test/utils/CrashReporterTest.java

```diff
@@ -162,6 +165,31 @@ public void testSharedPreferencesClearedOnLoggingFailed() {
 	}
 
 	@Test
+	public void testUnhandledCaughtExceptionSentOnCrashReportEnabled() {
+		Gson gson = new Gson();
+		String json = gson.toJson(exception);
+		editor.putString(CrashReporter.EXCEPTION_FOR_REPORT, json);
+		editor.commit();
```

Comment:
IMO it's better to use `CrashReporter.storeUnhandledException` in both tests instead.